ampd/src/main.rs

```diff
@@ -38,26 +41,41 @@ enum Output {
     Json,
 }
 
+#[derive(Debug, Subcommand, Valuable)]
+enum SubCommand {
+    /// Run the ampd daemon process (default)
+    Daemon,
+    /// Query the worker address
```

Comment:
it's not just a query right? if the address hasn't been set yet it'll actually run a keygen